### ISSUE TITLE
Ohran Viper (Not Destroying creature at end of combat step, after the damage it be given) #978

### ISSUE THREAD

**Issue by kelsioboy** [magarena/magarena#1362]
Magarena 1.89 problem returned. They are all cards that have "deals damage to a creature, destroy that creature to end of combat." no matter if the damage is combat or not. The cards for "blocks or becomes blocked by, destroy that creature at end of combat" are working normal.

**Comment by melvinzhang** [comment]
Duplicate of #978. Please reopen the existing issue instead of creating a new one if it is the same bug. Thanks
